src/accumulator/util.rs

```diff
@@ -291,6 +359,14 @@ mod tests {
     fn test_is_right_sibling() {
         assert!(super::is_right_sibling(0, 1));
     }
+
+    #[test]
+    fn test_remove_bit() {
+        // This should remove just one bit from the final number
+        // 15 = 1111, removing bit 3 makes it 111, that is 7
+        let res = super::remove_bit(11, 2);
+        assert_eq!(res, 7);
+    }
```

Comment:
It's easy to add more tests here.

Could you add some more? Just something formatted like

```
        // 1010 => 101
        let res = super::remove_bit(10, 0);         
        assert_eq!(res, 5);                                    

        // 1110 => 110                          
        let res = super::remove_bit(14, 1);
        assert_eq!(res, 6);
```

should be enough